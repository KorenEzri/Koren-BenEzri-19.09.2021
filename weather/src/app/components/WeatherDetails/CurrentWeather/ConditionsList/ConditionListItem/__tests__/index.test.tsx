import * as React from 'react';
import { render } from '@testing-library/react';

import { ConditionListItem } from '..';


describe('<ConditionListItem  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<ConditionListItem />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
