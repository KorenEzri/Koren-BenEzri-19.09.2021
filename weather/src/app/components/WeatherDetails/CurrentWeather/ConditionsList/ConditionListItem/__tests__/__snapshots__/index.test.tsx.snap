// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConditionListItem  /> should match snapshot 1`] = `
.c0 {
  margin-right: 20px;
  margin-bottom: 20px;
  padding: 20px;
  background-color: #ffffff;
  border-radius: 12px;
  width: 240px;
  height: 220px;
  text-align: center;
  -webkit-box-shadow: 3px 5px 7px -4px rgba(0,0,0,0.57);
  box-shadow: 3px 5px 7px -4px rgba(0,0,0,0.57);
}

.c1 {
  font-weight: bold;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
}

<div
  class="c0"
>
  <div
    class="c1"
  />
  <div
    class="ConditionListItem__ConditionListItemData-sc-10b7pfs-2"
  >
    <p />
  </div>
</div>
`;
